src/Components/SideBar/SideBar.jsx

```diff
@@ -88,12 +88,12 @@ class SideBar extends Component {
   render() {
     const { user, display = true } = this.props;
     return (
-      <nav style={{ display: display ? 'flex' : null }}>
+      <div className="sidenav" style={{ display: display ? 'flex' : null }}>
```

Comment:
Did you talk with Karl about this change because I can see that it is affecting his work?

Reply:
Yes! he is the one who allowed me to change it